src/OrchardCore/OrchardCore.ContentManagement.Abstractions/ContentElement.cs

```diff
@@ -14,11 +14,11 @@ public class ContentElement : IContent
     private Dictionary<string, ContentElement> _elements;
     private JsonDynamicObject _dynamicObject;
 
-    internal ContentElement() : this([])
+    protected ContentElement() : this([])
```

Comment:
Because it was protected before merging the references PR. Protected will allow us to continue to be able to inherit from it and does not break anyone